tests/numpy/test_misc.py

```diff
@@ -121,12 +121,102 @@ def test_all(nps_app_inst):
     assert nps.all(false_float).dtype is np.bool
 
 
+def test_alltrue(nps_app_inst):
```

Comment:
Is `alltrue` the same as `all`? If so, I think it's okay to test `all` and `alltrue` in the same test.

Reply:
Yes, I think `all` replaced `alltrue` in NumPy, but both are still supported. I'll combine their tests.